resources/views/includes/sidebar.blade.php

```diff
@@ -20,12 +21,13 @@
                 </a>
             </li>
             <li>
-                <a href="">
+                <a href="/friends">
```

Comment:
If this is leading to the search friends page, should this be '/search-friends' referring to the blade?